main.go

```diff
@@ -172,6 +172,18 @@ func fail(format string, v ...interface{}) {
 	os.Exit(1)
 }
 
+func findIosTargetPathComponent(target string, configuration string, cordovaVersion string) string {
+	if cordovaVersion != "" && cordovaVersion[0:1] >= "7" {
```

Comment:
This creates another issue, because some people use the "latest" as a version, which can use the late 12.0.0 version. But because latest is != "" but not a > 7 since its not a number, then this conditional falls to the old way. This caused me tons of hours of head of pain to find the root cause of why the build output was not finding any compiled ipa files.